ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -117,6 +117,67 @@ presubmits:
           name: prow-job-cluster-launch-installer-e2e
         name: job-definition
     trigger: ((?m)^/test( all| e2e-aws-all),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-libvirt
+    decorate: true
+    name: pull-ci-openshift-installer-master-e2e-libvirt
+    optional: true
+    rerun_command: /test e2e-libvirt
+    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-libvirt-cluster-profile
+        - --target=e2e-libvirt
+        - --template=/usr/local/e2e-libvirt
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-installer-master.yaml
+              name: ci-operator-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-libvirt
+        - name: TEST_COMMAND
+          value: TEST_FOCUS='Secrets should be consumable from pods in volume with
+            defaultMode set' run-tests
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-libvirt-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/e2e-libvirt
+          name: job-definition
+          subPath: cluster-launch-installer-e2e.yaml
```

Comment:
@stevekuznetsov  this should be `nested-libvirt-installer-e2e.yaml` right? this is the generated job